app/src/main/kotlin/com/mapbox/vision/teaser/recorder/RecorderFragment.kt

```diff
@@ -46,20 +62,39 @@ class RecorderFragment : Fragment() {
         initStopRecordButton()
     }
 
+    private fun initStopRecordButton() {
+        stop_record.setOnClickListener {
+            VisionManager.stopRecording()
+            requireActivity().onBackPressed()
+        }
+    }
+
     override fun onResume() {
         super.onResume()
+        onResumeARManager()
```

Comment:
We had quite a lot of discussion with @yunikkk before and AFAIR we use generally `Ar` instead of `AR` everywhere.
```suggestion
        onResumeArManager()
```

Reply:
fixed